strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/AbstractRepositoryProvider.java

```diff
@@ -290,20 +288,19 @@ protected Predicate createPredicate(String storageId,
                                         String repositoryId,
                                         Predicate predicate)
     {
-        Predicate result = Predicate.of(ExpOperator.EQ.of("storageId",
-                                                          storageId))
-                                    .and(Predicate.of(ExpOperator.EQ.of("repositoryId",
-                                                                        repositoryId)));
+        Predicate result = Predicate.of(ExpOperator.EQ.of("storageId", storageId))
```

Comment:
@carlspring this formatting seems not like you always reccomend with every parameter on new line :)

Reply:
Yeah... You're right. I just thought I'd move them on one line, because they're shorter than a line and are simply a key-value pair, but I can restore it, if you like.